test/zdtm/lib/test.c

```diff
@@ -239,34 +242,36 @@ void test_init(int argc, char **argv)
 		exit(1);
 	}
 
-	val = getenv("ZDTM_GROUPS");
-	if (val) {
-		char *tok = NULL;
-		unsigned int size = 0, groups[NGROUPS_MAX];
-
-		tok = strtok(val, " ");
-		while (tok) {
-			size++;
-			groups[size - 1] = atoi(tok);
-			tok = strtok(NULL, " ");
+	if (!uid) {
```

Comment:
imho: It is better to pass ZDTM_ROOTLESS from zdtm.py.

Reply:
Fixed as suggested.